File: Frontend/library-app/src/models/SuccessPaymentRequest.ts
class SuccessPaymentRequest {
  paymentKey: string;
  orderId: string;
  amount: number;
  constructor(paymentKey: string, orderId: string, amount: number) {
    this.paymentKey = paymentKey;
    this.orderId = orderId;
    this.amount = amount;
  }
}
export default SuccessPaymentRequest;

File: src/main/java/com/reactlibraryproject/springbootlibrary/RequestModels/SuccessPaymentRequest.java
package com.reactlibraryproject.springbootlibrary.RequestModels;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class SuccessPaymentRequest {
    private String paymentKey;
    private int amount;
    private String orderId;
}
